saleor/account/views.py

```diff
@@ -156,3 +158,20 @@ def account_delete_confirm(request, token):
 
     return TemplateResponse(
         request, 'account/account_delete_prompt.html')
+
+
+@login_required
+def email_edit(request):
+    form = EmailChangeForm(request, data=request.POST or None)
+
+    if request.method == 'POST':
+        if form.is_valid():
+            form.clean_email()
+            messages.success(request, pgettext(
+                'Storefront message', 'Email successfully changed.'))
```

Comment:
Is this correct? Is there code that actually changes the email address?

Reply:
@patrys It will be as you saying. Firstly we will be sending email to the old address. I'm making changes in code according to @Pacu suggestions, so it will be completely different. I'm was made this PR because I didn't know how to resolve this issue :)